src/main/java/org/sagebionetworks/bridge/services/AccountService.java

```diff
@@ -302,10 +303,13 @@ public void updateAccount(Account account) {
         account.setOrgMembership(persistedAccount.getOrgMembership());
         // Update modifiedOn.
         account.setModifiedOn(DateUtils.getCurrentDateTime());
+        account.setAdmin(persistedAccount.isAdmin());
```

Comment:
You will need to duplicate the above logic until this flag is set by the particular API being called.